src/components/ChatRoomPage/ChatInput.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import { useRecoilState, useRecoilValue } from 'recoil';
+import { currentUserState, allChatRoomsInfoState } from '../../state/atom';
+import styled from 'styled-components';
+import { IoIosSend } from 'react-icons/io';
+import { IChat, IUser } from '../../interface/interface';
+
+type ChatInputProps = {
+  friendInfo: IUser;
+  chatList: IChat[];
+};
+
+const ChatInput = ({ friendInfo, chatList }: ChatInputProps) => {
+  const currentUser = useRecoilValue(currentUserState);
+  const [text, setText] = useState('');
+  const [newChatList, setNewChatList] = useState(chatList);
+  const [allChatRoomsInfo, setAllChatRoomsInfo] = useRecoilState(
+    allChatRoomsInfoState
+  );
+
+  const addChat = () => {
+    setNewChatList([
+      ...newChatList,
+      { userId: currentUser.userId, content: text, time: currentTime() },
+    ]);
+  };
+
+  const currentTime = () => {
+    let hours = ('0' + new Date().getHours()).slice(-2);
+    let minutes = ('0' + new Date().getMinutes()).slice(-2);
+
+    return hours + ':' + minutes;
+  };
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    if (!text.trim().length) {
+      return;
+    }
+
+    addChat();
+
+    setText('');
```

Comment:
```suggestion
    if (text.trim().length) {
      addChat();
      setText('');
    }
```
여기는 이렇게 작성해도 무방할 것 같네요!